cli/src/config/types.ts

```diff
@@ -110,7 +110,7 @@ export interface ProviderConfig {
 	id: string
 	provider: ProviderName
 	// Provider-specific fields
-	[key: string]: any
+	[key: string]: unknown
```

Comment:
I wonder if it's too much work, but we could make a zod schema which is a union of each specific provider and then use that as the type here, as well as reuse the schema for validation.

Reply:
Yeah that definitely sounds like something we want at some point